ocs_ci/utility/environment_check.py

```diff
@@ -93,9 +93,11 @@ def assign_get_values(
 
     for item in items:
         ns = item.get('metadata', {}).get('namespace')
+        if item.get('kind') == constants.PV:
+            ns = item.get('spec').get('claimRef').get('namespace')
```

Comment:
I think it will be better to change it to this: `ns = item.get('spec', {}).get('claimRef', {}).get('namespace')`

Reply:
+1, will do

Reply:
I don't think that this is useful here. There needs to be returned `namespace` value from a given place in the dictionary. This way if some of the keys is not present, the fail will be silent and it will fail on `.get('namespace')` because there will be no 'namespace' key in empty dictionary.
We will not know which key was missing (if `spec` or `claimRef`) because on one or two of those steps was returned {} without error and Python won't provide information what key was missing right away.

Reply:
reverted